Mail/Views/Thread List/ThreadListHeader.swift

```diff
@@ -120,7 +131,7 @@ struct ThreadListHeader: View {
             }
         }
         .padding(.top, value: .small)
-        .padding([.leading, .trailing, .bottom], value: .medium)
+        .padding([.leading, .trailing, .bottom], value: .large)
```

Comment:
Why do we want to increase the padding? It doesn't look really nice with a bigger padding